tests/ZendTest/Mvc/Controller/Plugin/FilePostRedirectGetTest.php

```diff
@@ -324,7 +324,7 @@ public function testFieldsetAmountInFormEqualsFieldsetsInInputFilter()
 
         $this->assertEquals($params, $prgResult);
         $this->assertCount(count($params['links']),  $form->get('links')->getFieldsets());
-        $this->assertCount(count( $form->get('links')->getFieldsets()), $form->getInputFilter()->get('links')->getInputs());
+        $this->assertCount(count( $form->get('links')->getFieldsets()), $form->getInputFilter()->get('links')->getValidInput());  // #6497
```

Comment:
This is a change in behavior. Can you quantify how this might affect existing applications, please?